cmd/infra/azure/create.go

```diff
@@ -184,7 +184,7 @@ func (o *CreateInfraOptions) Run(ctx context.Context, l logr.Logger) (*CreateInf
 		// Extract network security group name
 		if vnet.Properties.Subnets[0].Properties.NetworkSecurityGroup != nil && vnet.Properties.Subnets[0].Properties.NetworkSecurityGroup.ID != nil {
 			result.SecurityGroupID = *vnet.Properties.Subnets[0].Properties.NetworkSecurityGroup.ID
-			securityGroupName, err := azureutil.GetNetworkSecurityGroupNameFromNetworkSecurityGroupID(*vnet.Properties.Subnets[0].Properties.NetworkSecurityGroup.ID)
+			securityGroupName, _, err := azureutil.GetNSGNameAndRGFromNSGID(*vnet.Properties.Subnets[0].Properties.NetworkSecurityGroup.ID)
```

Comment:
nit: I get why, but having to learn a bunch of acronyms is not ideal. Maybe `NameAndResourceGroupForNetworkSecurityGroupID`? We write this once, read it many times

Reply:
Yeah that works. I try not to put things in acronyms as I can and be verbose instead :)